src/button/styles/button.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
+	border-radius: 5px;
+	color: var(--button-color);
+	cursor: pointer;
+	font: var(--base-font);
+	font-weight: bold;
+	display: inline-block;
+	padding: 0.75em 1.25em;
+	transition: all 0.3s ease;
+}
+
+.button:hover,
+.button:focus,
+.button[aria-pressed="true"] {
+	background-color: var(--button-hover-bg);
+	border-color: var(--button-hover-bg);
+	outline: none;
+}
+
+.button[disabled],
+.button[disabled]:hover {
+	background-color: var(--button-disabled-bg);
+	border-color: var(--button-disabled-bg);
+	cursor: default;
+}
+
+.button[data-dojo-icon]:before {
```

Comment:
I'm not entirely sure what you mean by icon fonts coming with font size/weight etc. If you think it best, we could just drop the icon property and let users deal with it themselves, as you suggested. I think Torrey mentioned having an icon button though, so it might be nice to have an explicit property for it. One also wouldn't rule out the other -- a user could always ignore `icon` and add extra html to `content` if they wish.